worker-store-fs/src/main/java/com/hpe/caf/worker/datastore/fs/FileSystemDataStore.java

```diff
@@ -60,7 +72,8 @@ public FileSystemDataStore(final FileSystemDataStoreConfiguration config)
                 throw new DataStoreException("Cannot create data store directory", e);
             }
         }
-
+        healthcheck = new FileSystemDataStoreHealthcheck(dataStorePath);
+        healthcheckTimeout = Duration.ofSeconds(config.getDataDirHealthcheckTimeoutSeconds());
```

Comment:
I think that if the configuration isn't set that you will get a `NullPointerException` here.  Obviously this won't usually happen because you've adjusted the default configuration but as per the previous comment I'm not sure that you can always rely on that being used.

Reply:
See previous comment